app/controllers/api/account.php

```diff
@@ -3077,3 +3166,82 @@
 
         $response->dynamic($verificationDocument, Response::MODEL_TOKEN);
     });
+
+App::put('/v1/account/targets/:targetId/push')
+    ->desc('Update Account Target')
+    ->groups(['api', 'account'])
+    ->label('error', __DIR__ . '/../../views/general/error.phtml')
+    ->label('audits.event', 'target.update')
+    ->label('audits.resource', 'target/response.$id')
+    ->label('event', 'users.[userId].targets.[targetId].create')
+    ->label('scope', 'public')
+    ->label('docs', false)
+    ->param('targetId', '', new UID(), 'Target ID.')
+    ->param('identifier', '', new Text(Database::LENGTH_KEY), 'The target identifier (token, email, phone etc.)', true)
+    ->param('providerId', '', new UID(), 'Provider ID. Message will be sent to this target from the specified provider ID. If no provider ID is set the first setup provider will be used.', true)
```

Comment:
Let's remove this, clients shouldn't be able to change push providers for a single target

Reply:
removed this